channeld/channeld.c

```diff
@@ -2642,11 +2642,11 @@ static void handle_offer_htlc(struct peer *peer, const u8 *inmsg)
 		status_failed(STATUS_FAIL_MASTER_IO,
 			      "Duplicate HTLC %"PRIu64, peer->htlc_id);
 
-	/* FIXME: Fuzz the boundaries a bit to avoid probing? */
 	case CHANNEL_ERR_MAX_HTLC_VALUE_EXCEEDED:
-		failcode = WIRE_TEMPORARY_CHANNEL_FAILURE;
-		failmsg = tal_fmt(inmsg, "Maximum value exceeded");
+		failcode = WIRE_REQUIRED_CHANNEL_FEATURE_MISSING;
+		failmsg = tal_fmt(inmsg, "Mini mode: maximum value exceeded");
```

Comment:
https://github.com/ElementsProject/lightning/pull/3126/files#diff-3407b2ebe226795b44e9ba46bcebc4fbR2646-R2647

Reply:
**Mini** mode , is this the summer wumbo value ?